src/articles-controller.js

```diff
@@ -0,0 +1,27 @@
+import httpErrors from 'http-errors';
+import urls from '@educandu/educandu/utils/urls.js';
+import DocumentStore from '@educandu/educandu/stores/document-store.js';
+
+const { NotFound } = httpErrors;
+
+class ArticlesController {
+  static get inject() { return [DocumentStore]; }
+
+  constructor(documentStore) {
+    this.documentStore = documentStore;
+  }
+
+  registerPages(router) {
+    router.get('/articles/*', async (req, res) => {
+      const slug = req.params[0] || '';
+      const doc = await this.documentStore.findOne({ namespace: 'articles', slug });
```

Comment:
Small thing, but in order for us not to need to change this again when we remove namespaces, can we just find by slug here?

Reply:
Hmmm, not sure, this was the old signature how it was finding the documents... @ahelmberger

Reply:
Good idea :+1:

Reply:
Done